src/alphabets/dna.rs

```diff
@@ -34,7 +34,7 @@ pub fn n_alphabet() -> Alphabet {
 
 /// The IUPAC DNA alphabet (uppercase and lowercase).
 pub fn iupac_alphabet() -> Alphabet {
-    Alphabet::new(b"ACGTURYSWKMBDHVNacgturyswkmbdhvn")
+    Alphabet::new(b"ACGTRYSWKMBDHVNZacgtryswkmbdhvnz")
```

Comment:
I originally added Z rather than replace U with it and received the following error:

> error[E0277]: the trait bound `&[u8; 34]: std::iter::Iterator` is not satisfied
>   --> src/alphabets/dna.rs:37:5
>    |
> 37 |     Alphabet::new(b"ACGTURYSWKMBDHVNZacgturyswkmbdhvnz")
>    |     ^^^^^^^^^^^^^ the trait `std::iter::Iterator` is not implemented for `&[u8; 34]`
>    |
>    = note: `&[u8; 34]` is not an iterator; maybe try calling `.iter()` or a similar method
>    = note: required because of the requirements on the impl of `std::iter::IntoIterator` for `&[u8; 34]`
>    = note: required because of the requirements on the impl of `utils::text::IntoTextIterator<'_>` for `&[u8; 34]`
>    = note: required by `alphabets::Alphabet::new`

With that error, I thought it was best to do the replacement since U is not a valid character in the DNA alphabet anyway.